builtin/mainmenu/settings/components.lua

```diff
@@ -235,6 +235,15 @@ function make.path(setting)
 	}
 end
 
+if PLATFORM == "Android" then
```

Comment:
I wonder if all path settings should also be hidden on Android as well

Reply:
I think it's fine to keep them, they'll be hidden in the advanced settings anyway.